### PR TITLE
gives the clown a headset with a clown megaphone loudmode

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

gives the clown a headset with a clown megaphone loudmode

## Why It's Good For The Game

Because people steal the megaphones all the time, harder to steal a headset.

## Changelog
:cl:
add: the clowns headset
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -230,6 +230,14 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	icon_state = "srv_headset"
 	keyslot = new /obj/item/encryptionkey/headset_service
 
+/obj/item/radio/headset/headset_clown
+	name = "clown's headset"
+	desc = "A headset for the clown. Finally. A megaphone you can't take away."
```
Comment (ghost) [inline code/game/objects/items/devices/radio/headset.dm:235]: grammar check grammar check

**Finally, a megaphone you can't take away.** would flow better imo

**Comment by dapnee** [comment]
I hate loud mode and this just continues large messages in chat that don’t contribute to anything.

File: code/game/objects/items/devices/radio/headset.dm (hunk 1 above)
Comment (Seris02) [reply code/game/objects/items/devices/radio/headset.dm:235]: screee I purposefully put it like that

**Comment by Ghommie** [comment]
Redundant, there is already a clown megaphone in the teathral clown/mime room. It could be added as backend but it may as well become a forgotten relic.
By the by, tgstation also had a similar PR a while ago that went closed, yet this one is implemented better than it.

**Comment by Ghommie** [comment]
Make this headset backend (not available without adminspawn) for now and it'll be merged.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**